doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -113,12 +134,13 @@ Proposals (WK): Change title of this section to 2.3 Architectural Requirements;
 
 | Ref # | sub-category | Description |
 |----|------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `req.lcm.gen.01`	| General | The Architecture **shall** support zero downtime expansion/change of physical capacity (compute hosts, storage increase/replacement) |
-| `req.lcm.gen.02` | General	| The Architecture **shall** support plug-ins to manage deployment of workload to multiple sites and various technology stack. Multi-cloud support through hybrid, migrations, etc. |
-| `req.lcm.gen.03` | General |	The Architecture **shall** provide OpenStack Ironic Bare Metal provision to support VIM Instance provisioning (bootstrap), Management System, KVM host provisioning and Kubernetes host provisioning |
-| `req.lcm.adp.01` | Automated deployment | The Architecture **shall** allow the “cookie cutter” automated deployment to multiple sites. |
-| `req.lcm.adp.02` | Automated deployment | The Architecture **must** enable automated and incremental upgrade of the various software components in a manner that still allows tenant workloads to continue to operate. |
-| `req.lcm.cid.02` | CI/CD | The Architecture **shall** support integrating with CI/CD Toolchain for NFVI and VIM components Automation |
+| `req.lcm.gen.01`	| General | The Architecture **should** support zero downtime expansion/change of physical capacity (compute hosts, storage increase/replacement) |
```

Comment:
That is a **must** requirement from my perspective.

Reply:
agree, changed